test/unit/widgets/peoplePicker.test.ts

```diff
@@ -5,21 +5,112 @@ import {
   PeoplePicker,
   GroupPicker,
   Group,
-  GroupBuilder
+  GroupBuilder,
+  Person,
+  findAddressBook,
+  createNewGroup
 } from '../../../src/widgets/peoplePicker'
+import ns from '../../../src/ns'
 jest.mock('rdflib')
 jest.mock('solid-auth-client')
-// jest.mock('../../../src/store')
+const kb = require('../../../src/store')
+const fetcher = kb.fetcher
 
 const dom = new JSDOM('<!DOCTYPE html><p>Hello world</p>').window.document
 const element = dom.createElement('div')
 
+describe('FindAddressBook', () => {
+  it('exists', () => {
+    expect(findAddressBook).toBeInstanceOf(Function)
```

Comment:
Do not need tests that check that an imported function exists - TypeScript does this for us.

If anything, we might want to write these tests to make sure that the functionality are exposed through the public API, especially for aliases. But as a rule of thumb, you do not need to test for the existence of functionality, since TypeScript handles this for us.

Reply:
Thank you.  I will remove this one.

Reply:
I disagree, see https://github.com/solid/solid-ui/blob/master/src/widgets/index.js#L23.

Reply:
Let me try it out and test what @megoth said, please don't remove it for now.

Reply:
Again, as a rule of thumb. The example you refer to makes sense to have a unit test that test that the API we export is as expected.

Than again, I would refactor the code that you linked to so that we are explicit about which things we import and export, instead of simply smashing all methods from existing modules into one object using `Object.assign`. Then we would not need unit tests, as TypeScript would ensure that these methods exist.

Reply:
Discussion continued in https://github.com/solid/solid-ui/pull/218

Reply:
Even if you refactor widgets/index, then widgets/peoplePicker needs its own unit tests.
I'm sure that somewhere in solid-panes there is some code that would break if we would merge #218, but that's not how unit tests work. They should test the unit (widgets/peoplePicker) in isolation without relying on how other files use what it exports.

Reply:
So let's *not* remove any of the "exists" tests, please.

Reply:
Answered in #218

Reply:
:+1: